wordclock/AddressableLedStrip.c

```diff
@@ -486,16 +486,22 @@ static void FilterRainbow(void)
 
 void AlsApplyTextEffect(ETextEffect filter)
```

Comment:
Haha toch de request van Tim uitgevoerd

Reply:
Yep, was vroeg op en had ff zin in beetje coden...